src/js/oTable.js

```diff
@@ -0,0 +1,182 @@
+/* global exports, document, HTMLElement, CustomEvent, setImmediate */
+
+// TODO: Use Dom-Deletgate -- https://www.npmjs.com/package/dom-delegate
+
+function OTable(rootEl) {
+    const sortByColumn = function(columnIndex) {
```

Comment:
I think, if it's a private function, the norm is to put it in the prototype but prefixed with an underscore and mark it private in jsdoc